### ISSUE TITLE
项目运行

### ISSUE THREAD

**Issue by GOUAIAI** [hkchengrex/davis2016-evaluation#1]
您好，请问一下这个项目该怎么运行呢，测试DAVIS2016的话

**Comment by hkchengrex** [comment]
There is an example command in the readme file. Does that not work for you?

**Comment by wanglewoba** [comment]
跑出来的结果全都返回1.0

**Comment by GOUAIAI** [comment]
跑出来的结果全都返回1.0，要么就是0

**Comment by DUT-CSJ** [comment]
> 跑出来的结果全都返回1.0，要么就是0

yes, I use the ground truth to evaluate, the result is all 1.0, but if I use prediction from precomputed results, the result is all 0.0.
I want to know how to solve this problem.
thanks

**Comment by hkchengrex** [comment]
@DUT-CSJ Are the precomputed results in the 0/1 format or 0~255 format as mentioned in the readme?

**Comment by DUT-CSJ** [comment]
@hkchengrex I use 0~255 format to evaluate, my precomputed results only have two value (0 and 255), I think my format is right to be evaluated. 
Thanks for your reply. I have used two functions from official script to write a evaluation code and get a very close result with official result. Though I have not yet used your code to reach the right number, I solve my problem anyway. I think I do not get the right way to use your code.
Thank you very much.

**Comment by jiaxi0030** [comment]
![image](https://github.com/hkchengrex/davis2016-evaluation/assets/92349729/870db109-3209-46a9-87d2-4780dcf2f803)
请问返回这样是为什么呀？测试好多遍都是这样

**Comment by DUT-CSJ** [comment]
> ![image](https://user-images.githubusercontent.com/92349729/238638086-870db109-3209-46a9-87d2-4780dcf2f803.png) 请问返回这样是为什么呀？测试好多遍都是这样

Try to use this repo(https://github.com/isyangshu/AMC-Net/tree/main/EVALVOS) which can get right results.

**Comment by jiaxi0030** [comment]
thank you&nbsp;



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Shijie ***@***.***&gt;; 
发送时间: 2023年5月16日(星期二) 晚上8:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [hkchengrex/davis2016-evaluation] 项目运行 (Issue #1)





  
 请问返回这样是为什么呀？测试好多遍都是这样
  
Try to use this repo(https://github.com/isyangshu/AMC-Net/tree/main/EVALVOS) which can get right results.
